app/src/main/res/values-eu/strings.xml

```diff
@@ -0,0 +1,90 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+  French Revolutionary Calendar Android Widget
+  Copyright (C) 2011 - 2017 Carmen Alvarez
+  
+  This program is free software; you can redistribute it and/or
+  modify it under the terms of the GNU General Public License
+  as published by the Free Software Foundation; either version 3
+  of the License, or (at your option) any later version.
+  
+  This program is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU General Public License for more details.
+  
+  You should have received a copy of the GNU General Public License
+  along with this program; if not, see <http://www.gnu.org/licenses/>.
+
+-->
+<resources>
+
+    <string name="app_name">Egutegi frantsesa</string>
+    <string name="app_name_wide">Egutegi frantsesa (horizontala)</string>
+    <string name="app_name_narrow">Egutegi frantsesa (bertikala)</string>
+    <string name="app_name_minimalist">Egutegi frantsesa (minimalista)</string>
+    <string name="app_full_name">Egutegi frantsesa</string>
+    <string name="description">Uneko data eta ordua Egutegi errepublikar frantsesean erakusten duen widgeta.</string>
+    <string name="credits">Kredituak:</string>
+    <string name="wikipedia">* Wikipediako artikulua: http://eu.wikipedia.org/wiki/Egutegi_errepublikanoa</string>
+    <string name="equinox_tool">* AstroLib C++ programa, Mark Hussek egina, urte bakoitzeko abuztuko lehen eguna zehazteko: http://mhuss.com/AstroLib/</string>
```

Comment:
Does `abuztuko` refer to August or to autumn?

Reply:
Good catch! My mistake, it's true ```abuztuko``` refers to August, and should be ```udazkeneko``` to refer to autumn.